deploy/terraform/bootstrap/sap_library/variables_local copy.tf

```
@@ -0,0 +1,6 @@
+locals {
+  environment          = lower(try(var.infrastructure.landscape, ""))
```

now it's called environment.
```suggestion
  environment          = lower(try(var.infrastructure.environment, ""))
```